src/router/projects.js

```diff
@@ -587,4 +587,55 @@ export const MY_PROJECT_VIEWS = [
       },
     ],
   },
+  {
+    path: ':id/institutions/',
+    name: 'MyProjectInstitutionsView',
+    meta: {
+      itemName: 'institutions',
+      breadcrumbs: [
+        {
+          label: 'institutions',
+          to: 'MyProjectInstitutionsView',
+        },
+      ],
+    },
+    component: () =>
+      import(
+        /* webpackChunkName: "unpublished" */ '@/views/project/institutions/ListView.vue'
+      ),
+  },
+  {
+    path: ':id/institutions/create',
+    name: 'MyProjectInstitutionsCreateView',
+    meta: {
+      itemName: 'institutions',
+      breadcrumbs: [
+        {
+          label: 'institutions',
+          to: 'MyProjectInstitutionsView',
+        },
+      ],
+    },
+    component: () =>
+      import(
+        /* webpackChunkName: "unpublished" */ '@/views/project/institutions/CreateView.vue'
+      ),
+  },
+  {
+    path: ':id/duplication/request',
+    name: 'MyProjectDuplicationRequestView',
```

Comment:
I think you meant to put the `MyProjectDuplicationRequestView` in another commit. The `RequestView.vue` is not in this PR.